modules/javafx.graphics/src/main/native-glass/mac/GlassViewDelegate.m

```diff
@@ -1333,4 +1333,9 @@ - (GlassAccessible*)getAccessible
     return (GlassAccessible*)jlong_to_ptr(accessible);
 }
 
+- (NSEvent*)lastEvent
```

Comment:
could there be side effects with exposing this field?
concurrency issues?

Reply:
The native toolkits are single-threaded. Anyway, I chose to refactor this a bit, removing the exposed `lastEvent` field and instead providing the method `performWindowDrag` that is called instead. The logic to supply the `lastEvent` is now contained in `GlassViewDelegate`.